src/mkdocstrings/templates/python/material/attribute.html

```diff
@@ -2,10 +2,14 @@
 
   <div class="doc doc-object doc-attribute">
 
-    {% if not root or config.show_root_heading %}
+    {% if not root or config.show_root_heading%}
```

Comment:
Nitpick: please re-add the space to reduce diff and stay consistent :stuck_out_tongue: 

Same remark in every other file!

Reply:
Oops, good catch! I'll fix this.